rpcs3/stb_image.cpp

```diff
@@ -1,9 +1,4 @@
 #include "stdafx.h"
-
-extern "C"
-{
-#pragma warning(push, 0)
+// Defines STB_IMAGE_IMPLEMENTATION *once* for stb_image.h includes (Should this be placed somewhere else?)
+#define STB_IMAGE_IMPLEMENTATION
```

Comment:
Same thing here
stb_image asks for this to be defined *once* before a include is made